crates/nu-cli/src/commands/to_html.rs

```diff
@@ -6,7 +6,104 @@ use nu_errors::ShellError;
 use nu_protocol::{Primitive, ReturnSuccess, Signature, SyntaxShape, UntaggedValue, Value};
 use nu_source::{AnchorLocation, Tagged};
 use regex::Regex;
+use rust_embed::RustEmbed;
+use serde::{Deserialize, Serialize};
+use std::borrow::Cow;
 use std::collections::HashMap;
+use std::error::Error;
+use std::io::Read;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct HtmlThemes {
+    themes: Vec<HtmlTheme>,
+}
+
+#[allow(non_snake_case)]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct HtmlTheme {
+    name: String,
+    black: String,
+    red: String,
+    green: String,
+    yellow: String,
+    blue: String,
+    purple: String,
+    cyan: String,
+    white: String,
+    brightBlack: String,
+    brightRed: String,
+    brightGreen: String,
+    brightYellow: String,
+    brightBlue: String,
+    brightPurple: String,
+    brightCyan: String,
+    brightWhite: String,
+    background: String,
+    foreground: String,
+}
+
+impl Default for HtmlThemes {
+    fn default() -> Self {
+        HtmlThemes {
+            themes: vec![HtmlTheme::default()],
+        }
+    }
+}
+
+// impl Default for HtmlTheme {
```

Comment:
You can just delete this. If we want to update the theme, should be easy enough

Reply:
oh, ya. that was a last minute thing. i was defaulting to campbell but then it made every test fail, so i commented it out to test the original and forgot to delete it. i'll remove it in the morning. thx.